clean_code/player.py

```diff
@@ -2,141 +2,144 @@
 import pygame
 from tkinter.filedialog import *
 from tkinter import *
+from enum import Enum
 pygame.init()
-global list1
-global pausing
-global list_index
-global SONG_END
-global text1
-global label1
-class FrameApp(Frame):
+
+class ButtonsActions(Enum):
+        ADD_TO_LIST = "ADD TO LIST"
+        PLAY_SONG = "PLAY SONG"
+        PAUSE_UNPAUSE = "PAUSE/UNPAUSE"
+        PREVIOUS_SONG = "PREVIOUS SONG"
+        NEXT_SONG = "NEXT SONG" 
+
+class SongPlayer:
+    def __init__(self):
+        self.list = list()
+        self.pausing = False
+        self.list_index = 0
+        self.SONG_END = pygame.USEREVENT + 1
+    
+    def add(self):
+        try:
+            directory = askopenfilenames()
+            
+            for song_dir in directory:
+                self.list.append(song_dir)
+        except:
+            pass
+    
+    def data(self, song):
+        try:
+            song = EasyID3(song)
+            song_data = "Now playing: Nr:" + str(self.list_index + 1) + " " + \
+                        str(song['title']) + " - " + str(song['artist'])
+            return song_data
+        except:
```

Comment:
Relevant for all occurrences, it's best if you specify the exception you're catching and if you do something in the except clause (at least a print)